python/tvm/relay/qnn/op/legalizations.py

```diff
@@ -20,8 +20,8 @@
 
 import tvm
 from tvm import relay
+import numpy as np
```

Comment:
A minor problem: Lint reports `python/tvm/relay/qnn/op/legalizations.py:23: [C0411(wrong-import-order), ] third party import "import numpy as np" should be placed before "import tvm"` on my platform. I have no idea why this PR can pass CI.

Reply:
I'm not seeing this and CI doesn't appear to be failing on any other patches, can you post what command you're using to perform the linting?

Reply:
I run `./tests/scripts/task_lint.sh` directly from the repository root.

My `pylint` version:
```
pylint 2.4.4
astroid 2.3.3
Python 3.7.0 (default, Mar 12 2019, 20:10:09)
[GCC 7.3.0]
```

Reply:
If you take a look here:
https://github.com/apache/incubator-tvm/blob/063ba63a4448432a1e17ec8b793152cd8bba7c2c/docker/install/ubuntu_install_python_package.sh#L24
CI runs with pylint 1.9.4 and I imagine that's where the difference is coming from.